### PR TITLE
Adding support for Athena

### PR DESCRIPTION
**Pull request by sean-garvey**
* Added Athena Task and tests
* Edited typos in README.md

### CODE REVIEW

**Comment by javibravo** [comment]
@sean-garvey,

Code looks good. Still missing something:

- Documentation. You should update README with the new type of task.
- Update LambdaCron policy. LambdaCron needs permissions on Athena to be able to execute queries: https://github.com/MediaMath/lambda-cron/blob/master/lambda_cron/template.cfn.yml#L63

**Review by javibravo — COMMENTED** [review]
Great job @sean-garvey !

I will wait to approve until we are able to test in AWS.

File: lambda_cron/schema.json [hunk 1]
```diff
@@ -47,19 +47,9 @@
...
-                  "type": "object",
-                  "properties": {
-                    "EncryptionOption": {
-                      "type": "string",
-                      "enum": ["SSE-S3", "Sse-s3", "sse-s3", "SSE-KMS", "Sse-kms", "sse-kms", "CSE-KMS", "Cse-kms", "cse-kms"]
-                    }
-                  },
-                  "required": ["EncryptionOption"]
-                }
+                "OutputLocation": { "type": "string" }
```
Comment (javibravo) [inline lambda_cron/schema.json]: What about the options "ClientRequestToken" and "QueryExecutionContext": http://boto3.readthedocs.io/en/latest/reference/services/athena.html#Athena.Client.start_query_execution ?

They are not required but if the are used, is the validation ok ? It should allow to use any parameter in the function (start_query_execution) and force required ones.

File: lambda_cron/template.cfn.yml [hunk 2]
```diff
@@ -91,10 +91,16 @@ Resources:
             Action:
               - batch:SubmitJob
             Resource: ['*']
-            - Effect: Allow
+          - Effect: Allow
             Action:
               - athena:StartQueryExecution
             Resource: ['*']
+          - Effect: Allow
```
Comment (javibravo) [inline lambda_cron/template.cfn.yml]: This should replace https://github.com/MediaMath/lambda-cron/blob/4369b40902a3a2a160b7b0f99d166bfaa011bcb4/lambda_cron/template.cfn.yml#L77

It is redundant

**Comment by sean-garvey** [comment]
Javier and I have discussed whether or not we should validate AWS API parameters (as the API may evolve over time) and have decided to table to the topic for now.

**Comment by javibravo** [comment]
Tested in AWS, all ok. Feature ready to merge!

@sean-garvey ,
One more thing we forgot, we need to update the diagram in the README. To update it:
1. Load [lambda-cron-diagram.xml](./lambda-cron-diagram.xml) in https://www.draw.io/ (you don't need to sign up).
2. Update diagram.
3. Export diagram as PNG and save XML file.
4. Add both of them to the repo.

**Review by javibravo — APPROVED** [review]

**Pull request merged**